deploy-disconnected-registry/olm-sync.sh

```diff
@@ -97,6 +97,15 @@ function mirror() {
     sed -i '/^mountopt =.*/d' /etc/containers/storage.conf
     #######
 
+    ####### WORKAROUND: Fixes "Invalid GPG signature for image certified-operator-index"
+    ####### https://access.redhat.com/solutions/6542281
+    curl -s -o /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-isv https://www.redhat.com/security/data/55A34A82.txt
```

Comment:
Can we add this `55A34A82.txt` file to the repo? Otherwise this will fail in a disconnected environment

Reply:
Where in the repo?